Content.Server/Cargo/Systems/CargoSystem.Bounty.cs

```diff
@@ -300,6 +300,21 @@ public bool IsBountyComplete(EntityUid container, IEnumerable<CargoBountyItemEnt
         return IsBountyComplete(GetBountyEntities(container), entries, out bountyEntities);
     }
 
+    /// <summary>
+    /// Determines whether the <paramref name="entity"/> meets the criteria for the bounty <paramref name="entry"/>.
+    /// </summary>
+    /// <returns>true if <paramref name="entity"/> is a valid item for the bounty entry, otherwise false</returns>
+    public bool EntityCountsForBountyEntry(EntityUid entity, CargoBountyItemEntry entry)
```

Comment:
rename to IsBountyEntry or IsAllowedBountyEntry, or something you get me

Reply:
Yeah, I wasn't wild about the name either. Thanks for the suggestions!